### PR TITLE
addressing pysal/pysal#1145

### PR DESCRIPTION
**Pull request by jGaboardi**
addressing pysal/pysal#1145

### CODE REVIEW

**Comment by jGaboardi** [comment]
> Seems that testing against the libpysal github master branch is failing because of the "nyc_bikes_ct.csv" data set?

Good catch. Similar issues seem to be affecting [`splot`](https://travis-ci.org/pysal/splot), [`spreg`](https://travis-ci.org/pysal/spreg), and [`spvcm`](https://travis-ci.org/pysal/spvcm).

File: .travis.yml [hunk 1]
```diff
@@ -32,20 +32,25 @@ before_install:
 install:
   - conda install --yes pip
   - conda install --yes --file requirements.txt;
-  - pip install https://github.com/pysal/splot/archive/master.zip
+  - git clone https://github.com/pysal/splot.git;
```
Comment (weikang9009) [inline .travis.yml]: Not sure why `splot` is installed here for testing?

Comment (jGaboardi) [reply .travis.yml]: I thought that was a bit out of place, as well. Shall I see if this PR passes without `splot`?

Comment (jGaboardi) [reply .travis.yml]: travis passes without `splot`. not sure how that was in there to begin with.

**Pull request merged**